vault/manifest.yaml

```diff
@@ -0,0 +1,6 @@
+name: Vault
```

Comment:
Can we have the same format as this? 
name: vault
title: "Vault"

Reply:
Could not find this in any other manifest nor in the docs. 
Is that something new?

Added it now as proposed.